packages/api/src/controllers/playback.test.ts

```diff
@@ -143,7 +147,44 @@ describe("controllers/playback", () => {
 
     describe("for streams", () => {
       it("should return playback URLs for streams", async () => {
-        const res = await client.get(`/playback/${stream.playbackId}`);
+        client.jwtAuth = adminToken;
+        let res = await client.post(`/experiment`, {
+          name: "stream-pull-source",
+        });
+        expect(res.status).toBe(201);
+        res = await client.post(`/experiment/stream-pull-source/audience`, {
+          addUsers: [mockAdminUserInput.email],
+        });
+        expect(res.status).toBe(204);
+
+        res = await client.get(`/playback/${stream.playbackId}`);
+        expect(res.status).toBe(200);
+        await expect(res.json()).resolves.toMatchObject({
+          type: "live",
+          meta: {
+            live: 0,
+            source: [
+              {
+                hrn: "HLS (TS)",
+                type: "html5/application/vnd.apple.mpegurl",
+                url: `${ingest}/hls/${stream.playbackId}/index.m3u8`,
+              },
+              {
+                hrn: "WebRTC (H264)",
+                type: "html5/video/h264",
+                url: `${ingest}/webrtc/${stream.playbackId}`,
+              },
+              {
+                hrn: "FLV (H264)",
+                type: "video/x-flv",
+                url: `${ingest}/flv/${stream.playbackId}`,
```

Comment:
Is this the correct URL? Shouldn't it have some suffix like `.flv`?

btw. I tried it in the box and it's not playable, but maybe it's just the box...

Reply:
Yeah it's no suffix like webrtc e.g. this works: `https://livepeercdn.studio/flv/db15j36hse0i9lay`. The box seems to have some different nginx setup going on which stops that working. So we can fix that separately.